src/Components/Rating/Rating.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+function Rating(props) {
+  const { children } = props;
+
+  let roundedRating = Math.round(children);
+
+  switch (roundedRating) {
+    case 0:
+      roundedRating = '☆☆☆☆☆';
+      break;
+    case 1:
+      roundedRating = '★☆☆☆☆';
+      break;
+    case 2:
+      roundedRating = '★★☆☆☆';
+      break;
+    case 3:
+      roundedRating = '★★★☆☆';
+      break;
+    case 4:
+      roundedRating = '★★★★☆';
+      break;
+    case 5:
+      roundedRating = '★★★★★';
+      break;
+    default:
+      return 'N/A';
+  }
```

Comment:
This works but it's very manual, you could do it in better ways such as a loop or a string.repeat()